test/plugin_functional/test_suites/core_plugins/applications.js

```diff
@@ -32,6 +32,12 @@ export default function ({ getService, getPageObjects }) {
   const loadingScreenShown = () =>
     testSubjects.existOrFail('kbnLoadingMessage');
 
+  const getKibanaUrl = path => {
+    const host = process.env.TEST_KIBANA_HOST || 'localhost';
```

Comment:
I know it's silly, but generally prefer to avoid creating properly formatted strings when there's a perfectly good format function available:

```js
return url.format({
  protocol: 'http:',
  hostname: process.env.TEST_KIBANA_HOST || 'localhost',
  port: process.env.TEST_KIBANA_PORT || '5620',
  path
})
```

A lot of these formats have tricky little edge cases that pop up over time.